napari/_qt/_qapp_model/qactions/_file.py

```diff
@@ -0,0 +1,126 @@
+from typing import List
+
+from app_model.types import Action, KeyCode, KeyMod, StandardKeyBinding
+
+from ...._app_model.constants import CommandId, MenuGroup, MenuId
+from ....utils.translations import trans
+from ...qt_main_window import Window
+from ...qt_viewer import QtViewer
+
+
+def _restart(window: Window):
+    window._qt_window.restart()
+
+
+FILE_ACTIONS: List[Action] = [
```

Comment:
For consistency could this be `Q_FILE_ACTIONS`? See: https://github.com/napari/napari/pull/4977#discussion_r980615906

cc @DragaDoncila

Reply:
Done